### DISCUSSION TITLE
Provide ability to mark items as resource

### DISCUSSION THREAD

**Discussion by agolp** [fvtt-fria-ligan/forbidden-lands-foundry-vtt#278] [Ideas]
It would be great to be able to mark an item as a resource (and have a dice value next to it).

Currently there's a way to track food, water, torches and arrows/bullets depending on the game, but sometimes there are more resources we would like to track at my table (for instance, different kinds of arrows or spell ingredients).

Ideally we would be able to press the dice value next to the item to roll the resource dice and update the value depending on the result of the roll.

**Comment by nocturno709** [comment]
LOVE this idea ! We do the same at my table

**Comment by narukaioh** [comment]
The idea is interesting, but it looks like a house rule. The system tries to implement the rule as written.

**Reply by agolp** [reply]
Sorry, I just realised comments are stored as trees here. Just in case you didn't see it, my reply is below your comment.

**Comment by agolp** [comment]
You are right, using resources for spell ingredients is a common houserule. It's not a houserule for the different types of arrows (iron and wooden if you want to carry both). I admit implementing the feature for arrows only doesn't seem to make a lot of sense.

Another place it would help is for food or water carried by a mount (to reduce encumbrance). Currently, you need to add the Resource die back to your character to roll and then clear the values on your character sheet and move the items back to the mount everyday.
